pkg/pillar/dpcreconciler/linux.go

```diff
@@ -903,7 +907,12 @@ func (r *LinuxDpcReconciler) getIntendedL3Cfg(dpc types.DevicePortConfig) dg.Gra
 	}
 	intendedL3 := dg.New(graphArgs)
 	intendedL3.PutSubGraph(r.getIntendedAdapters(dpc))
-	intendedL3.PutSubGraph(r.getIntendedSrcIPRules(dpc))
+	// XXX comment out this ip rule, this prevents kubernetes pods communicate
```

Comment:
I will test what is the effect of not including these source IP rules. Maybe it is sufficient for EVE to set the source IP address and Linux will route it through the corresponding interface, i.e. without having to create and use a separate per-port routing table.